src/utils/metadataDelta/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './utils';
+export * from './types';
```

Comment:
This opens a whole new discussion on block-ingestor folder structure but there's `src/types` folder already and I'm not sure about exporting types from `~utils`.